strapi/strapi
# Fix: Webhook & lifecyles payload with populated relations

### What does it do?

Add an admin config variable to enable relations performance.

### Why is it needed?
We introduced a regression after the relations performance feature. 

We were not populating relation fields when creating/updating/... an entity, and now the content manager handles relations fields in a separate (and paginated) way.

A side effect of that is the webhook is not receiving populated relations, and this PR aims to provide a solution for it.

### How to test it?

- Create a webhook set to be triggered when an entity is created. 
- Create a new entity with relations.

You should see the relation fields to be something like: { count: <num-of-relations> }

In `examples/getstarted/config/admin.js` set relations.performance config to false:
Repeating the same steps should result in seeing relations fields to be populated.

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/14882
